prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1817,46 +1822,61 @@ static inline void rdm_ep_poll_ibv_cq(struct rxr_ep *ep,
 	efa_ep = container_of(ep->rdm_ep, struct efa_ep, util_ep.ep_fid);
 	efa_av = efa_ep->av;
 	efa_cq = container_of(ep->rdm_cq, struct efa_cq, util_cq.cq_fid);
+
 	for (i = 0; i < cqe_to_process; i++) {
-		ret = ibv_poll_cq(efa_cq->ibv_cq, 1, &ibv_wc);
+		if (i < 1) {
+			/* Call ibv_start_poll only once */
+			ret = ibv_start_poll(efa_cq->ibv_cq_ex, &poll_cq_attr);
```

Comment:
similarly, `ibv_start_poll` should be outside of loop.

For this function, I think we can get rid of `cqe_to_process`, just process until there is no CQ entries, e.g. `ibv_next_poll` return ENOENT.

So something like:

```
    err = ibv_start_poll();
    if (err)
         return;

   while(!err) {
       process_cq();

       err = ibv_next_poll();
    }

    ibv_stop_poll().
```